qp_target_gene/split_libraries/tests/test_split_libraries_fastq.py

```diff
@@ -69,8 +69,11 @@ def test_get_sample_names_by_run_prefix_error(self):
             f.write(MAPPING_FILE_2)
         self._clean_up_files.append(fp)
 
-        with self.assertRaises(ValueError):
+        with self.assertRaises(ValueError) as error:
```

Comment:
Not necessary to change, but asserRegexp can also check for the exact message you are expecting to see, or a regex, [see here](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaisesRegexp).